Source/Core/Core/BootManager.cpp

```diff
@@ -212,6 +212,11 @@ void ConfigCache::RestoreConfig(SConfig* config)
 
 static ConfigCache config_cache;
 
+void SetEmulationSpeedReset(bool value) 
```

Comment:
Looks like there's a trailing space here?
```suggestion
void SetEmulationSpeedReset(bool value)
```